warehouse/query-core/src/main/java/datawave/query/common/grouping/AverageAggregator.java

```diff
@@ -120,11 +120,7 @@ public BigDecimal getDivisor() {
     @Override
     public void aggregate(Attribute<?> value) {
         BigDecimal number;
-        try {
-            number = new BigDecimal(value.getData().toString());
-        } catch (Exception e) {
-            throw new IllegalArgumentException("Unable to calculate an average with non-numerical value '" + value.getData() + "'", e);
-        }
+        number = new BigDecimal(value.getData().toString());
```

Comment:
Do we want/need to catch the NumberFormatException that can be thrown here?

Reply:
Why not simply let it get passed up?  We were only turning it into an IllegalArgumentException anyway...

Reply:
As long as something above is catching it that's fine, just wanted to make sure it had been considered